apps/bundle-size/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../scripts/typescript/tsconfig.common.json",
```

Comment:
**suggestion:** we can use the new DX for this -> extending from `tsconfig.base.json`.

with that following could be removed

```diff
-"typeRoots": ["../../node_modules/@types"],
-"strict": true
```


also we won't build physical files from this so we can add `noEmit` directly
>📅 in future we'll hopefully use babel/esbuild for transpilation, thus TS will be used only for declaration files generation

Reply:
Done in b6df591 👍